app/models/liabilitycalculation/viewmodels/asd.sc

```diff
@@ -0,0 +1,38 @@
+import org.joda.time.format.DateTimeFormatter
+
+import java.time.{LocalDate, LocalDateTime, ZoneId, ZoneOffset}
+//import java.time.DateFormatter
+
+def getInYearCalculationRunDate(calculationTimestamp: String, taxYear: Int): Option[String] = {
+  //val dateFormatter = DateFormatter
+  val firstQuarterstart = LocalDate.parse(s"${taxYear - 1}-04-05")
+  val firstQuarterEnd = LocalDate.parse(s"${taxYear - 1}-07-05")
+  val SecondQuarterEnd = LocalDate.parse(s"${taxYear - 1}-10-05")
+  val thirdQuarterEnd = LocalDate.parse(s"${taxYear}-01-05")
+  val fourthQuarterEnd = LocalDate.parse(s"${taxYear}-04-05")
+  //2019-02-15T09:35:15.094Z
+   val df = java.time.format.DateTimeFormatter.ofPattern("yyyy-mm")
+  val formatter = java.time.Instant.parse(calculationTimestamp)
+  val date = LocalDateTime.ofInstant(formatter,ZoneId.of(ZoneOffset.UTC.getId)).toLocalDate
+  println(thirdQuarterEnd.getMonthValue+"!!!" +date.getMonthValue)
```

Comment:
I don't think we need this test file at all, pls remove; in general these need to be wrapped in the relevant unit test, so looks like something is missing